proto/cosmos/bank/v1beta1/bank.proto

```diff
@@ -82,4 +82,9 @@ message Metadata {
   // display indicates the suggested denom that should be
   // displayed in clients.
   string display = 4;
+  // name defines the name of the token (eg: Cosmos Atom)
+  string name = 5;
+  // symbol is the token symbol usually shown on exchanges (eg: ATOM). This can
+  // be the same as the display.
+  string symbol = 6; 
```

Comment:
Do we really need symbol? Can you give an example where `symbol` is not `display` upper/lower-cased?

Reply:
@jtremback thought here? I'm fine with either tbh

Reply:
BTC vs Bitcoin, ETH vs Ether, etc.